application/controllers/admin/DataEntry.php

```diff
@@ -614,12 +614,25 @@ public function editdata($subaction, $id, $surveyid)
                 }
 
                 $highlight = !$highlight;
-                $aDataentryoutput .= ">\n"
-                . "<td>"
-                . "\n";
+                $aDataentryoutput .= ">\n";
+                // First column (Question)
+                $aDataentryoutput .= "<td>";
                 $aDataentryoutput .= stripJavaScript($question);
-                $aDataentryoutput .= "</td>\n"
-                . "<td>\n";
+                $aDataentryoutput .= "</td>\n";
+                // Second column (Unseen)
+                $aDataentryoutput .= "<td style='width: 120px;'>\n";
+                // Only show the "Unseen" checkbox for questions.
+                if (!empty($fname['title'])) {
+                    $aDataentryoutput .= "<div class='checkbox'>\t"
+                        . "<input type='checkbox' name='{$fname['fieldname']}_unseen' id='{$fname['fieldname']}_unseen'"
+                        . (is_null($idrow[$fname['fieldname']]) ? " checked" : "")
+                        . ">"
+                        . "<label for='{$fname['fieldname']}_unseen'>" . gT("Unseen") . "</label>"
```

Comment:
Potential issue with subquestion with title == 'unseen'

Maybe use non alphanumeric character ? Using `:` maybe ? or `-` ? (boith seems allowed in name and in id)